packages/paste-website/src/pages/components/menu/index.mdx

```diff
@@ -229,6 +229,8 @@ To do so, either set `href` with a valid url to go to a new page, or set the `on
   </CalloutText>
 </Callout>
 
+Use the `variant` prop in `MenuItem` to use a default or destructive item.
```

Comment:
Suggestion: because we are only using the red to indicate that it is a destructive item, we should maybe add a recommendation that the text should be something that is commonly understood as destructive.